platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/UpdateNotification.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+public class UpdateNotification {
+    public static var showPush: Bool {
+        let result: Result<Bool, Error> = Core.instance.executeRequest(.getShowPushNotification)
+        switch result {
+        case .success(let value):
+            return value
+        case .failure(let error):
+            print("getShowPushNotification request failed: \(error.localizedDescription)")
+            return false
+        }
+    }
```

Comment:
Can be refactored as:
```suggestion
    public static var showPush: Bool {
       Core.instance
               .executeRequest(.getShowPushNotification)
               .inspectError { err in
                   Log.err(print("getShowPushNotification request failed: \(error.localizedDescription)"))
               }
               .unwrapOr(false)
      }
```

Same for the rest below.

Reply:
We don't have these extensions in `Core` :/ Do you think I should move them there?

Reply:
True, I missed that. Yeah it can be temporarily moved to Core to make use of it.

Reply:
Done!